kubetest/kind/kind.go

```diff
@@ -55,13 +58,21 @@ var (
 			"or %q (download a stable binary).", kindBinaryBuild, kindBinaryStable))
 )
 
-// Kind is an object the satisfies the deployer interface.
-type Kind struct {
+var (
+	kindBinaryStableHashes = map[string]string{
```

Comment:
We now persist the binary in `$HOME/.kubetest/kind/` and avoid redownloading it when the sha256sum matches the known hash for the platform's stable kind release.